splunk_eventgen/lib/plugins/output/httpevent.py

```diff
@@ -98,28 +98,37 @@ def updateConfig(self, config):
                         'outputMode httpevent but httpeventServers not specified for sample %s' % self._sample.name)
             # set default output mode to round robin
             if hasattr(self.config, 'httpeventOutputMode') and self.config.httpeventOutputMode:
-                self.httpeventoutputmode = config.httpeventOutputMode
+                self.httpeventoutputmode = self.config.httpeventOutputMode
             else:
                 if hasattr(self._sample, 'httpeventOutputMode') and self._sample.httpeventOutputMode:
                     self.httpeventoutputmode = self._sample.httpeventOutputMode
                 else:
                     self.httpeventoutputmode = 'roundrobin'
+
             if hasattr(self.config, 'httpeventMaxPayloadSize') and self.config.httpeventMaxPayloadSize:
                 self.httpeventmaxsize = self.config.httpeventMaxPayloadSize
             else:
                 if hasattr(self._sample, 'httpeventMaxPayloadSize') and self._sample.httpeventMaxPayloadSize:
                     self.httpeventmaxsize = self._sample.httpeventMaxPayloadSize
                 else:
                     self.httpeventmaxsize = 10000
+
+            if hasattr(self.config, 'httpeventAllowFailureCount') and self.config.httpeventAllowFailureCount:
+                self.httpeventAllowFailureCount = int(self.config.httpeventOutputMode)
```

Comment:
this should not be `httpeventOutputMode`.

Reply:
great catch